src/components/ChattingScreen.js

```diff
@@ -1,31 +1,125 @@
-		{ user: true, content: '상단에 프로필을 눌러서 발신자 변경하면 됩니당~' },
-		{ user: false, content: '안녕 은아' },
-		{ user: false, content: '뭐해 ?' },
-		{ user: true, content: '시원아 넌 최고의 팀장이야' },
-		{ user: false, content: '나도 아니까  ' },
-		{ user: false, content: '그만 말해줘도 돼' },
-		{ user: true, content: 'ㅠㅠ' },
-	];
-
-	return (
-		<Wrapper>
-			<Header></Header>
-			<MessageSender></MessageSender>
-		</Wrapper>
-	);
+  const EUNKO = 'https://img.techpowerup.org/200908/eun.png';
+  const COOL = 'https://img.techpowerup.org/200908/NjRiY2JjOGU5YzQz.png';
+
+  const MSGLIST = [
+    { user: true, content: '안녕하세요 12기 프론트엔드 개발자분들' },
+    { user: true, content: '저희의 대화를 마음껏 조작해보세요 💌' },
+    { user: true, content: '상단에 프로필을 눌러서 발신자 변경하면 됩니당~' },
+    { user: false, content: '안녕 은아' },
+    { user: false, content: '뭐해 ?' },
+    { user: true, content: '시원아 넌 최고의 팀장이야' },
+    { user: false, content: '나도 아니까  ' },
+    { user: false, content: '그만 말해줘도 돼' },
+    { user: true, content: 'ㅠㅠ' },
+  ];
+
+  const [message, setMessage] = useState(MSGLIST);
+  const [newMessage, setNewMessage] = useState({
+    user: false,
+    content: '',
+  });
+
+  const addNewChat = (e) => {
+    e.preventDefault();
+    if (newMessage.content == '') {
+      alert('type something!!');
+      return;
+    }
+    setMessage([...message, newMessage]);
+    setNewMessage({ ...newMessage, content: '' });
+  };
+
+  console.log(' new', newMessage.content);
+  console.log(' message', ...message);
+
+  const changeInputChat = (e) => {
+    const { name, value } = e.target; //name으로 구별, value는 값 자체
+    setNewMessage({
+      ...newMessage,
+      [name]: value,
+    });
+  };
+
+  const changeUser = (e) => {
+    return newMessage.user
+      ? setNewMessage({ user: false, content: newMessage.content })
+      : setNewMessage({ user: true, content: newMessage.content });
+  };
+
+  useEffect(() => {
+    window.scrollBy(1000, window.innerHeight);
+  }, [newMessage]);
+
+  return (
+    <Wrapper>
+      <Header
+        {...{ changeUser }}
+        user={newMessage.user ? '고은' : '정쿨'}
+        imgURL={newMessage.user ? EUNKO : COOL}
+      ></Header>
+      <MessageList>
+        {message.map((message, index) => {
+          if (message.user) {
+            return (
+              <ChatLine left>
+                <Profile src={EUNKO} width='50' height='50' />
+                <ChatBox left>{message.content}</ChatBox>
+              </ChatLine>
+            );
+          }
+          return (
+            <ChatLine>
+              <ChatBox>{message.content}</ChatBox>
+              <Profile src={COOL} width='50' height='50' />
+            </ChatLine>
+          );
+        })}
+      </MessageList>
+      <MessageSender
+        {...{ addNewChat }}
+        {...{ newMessage }}
+        {...{ changeInputChat }}
+      />
+    </Wrapper>
+  );
 }
 
-const Wrapper = styled.div``;
+const Wrapper = styled.div`
+  background-color: #abc7d1;
+`;
+
+const MessageList = styled.div`
```

Comment:
Messagesender 컴포넌트가 p`osition:fixed`로 설정되어있어서 메시지리스트를 추가해도 messagesender 컴포넌트에 가려져요ㅠㅠ,
 이 경우엔 메시지리스트를 감싸고 있는 컴포넌트에 패딩값을 줘서 해결하시면 됩니당 [참고](https://developer.mozilla.org/ko/docs/Web/CSS/padding)

```suggestion
const MessageList = styled.div`
	padding: 100px 10px;
```

Reply:
오오오오 감사합니다!!

Reply:
오오오오 감사합니다!!